src/components/Header/Notifications.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const Notifications = ({ plan , notifications, emptyNotificationText }) => {
+  let dataCount = [];
+  dataCount = notifications.length? dataCount['data-count'] = notifications.length:'';
```

Comment:
What is the type of _dataCount_??

It seems that you are mixing Array, Object, and String.

Also, in general, we could make the code better and clear with immutability (using `const` in place of `let`).

What do you think about this:

```js
const dataCountAttr = notifications.length ? { data-count: notifications.length } : {};
```